intg/src/main/java/org/apache/atlas/AtlasConfiguration.java

```diff
@@ -136,6 +136,8 @@ public enum AtlasConfiguration {
     ATLAS_INDEXSEARCH_ENABLE_JANUS_OPTIMISATION_FOR_LINEAGE("atlas.indexsearch.enable.janus.optimization.for.lineage", false),
     ATLAS_LINEAGE_ENABLE_CONNECTION_LINEAGE("atlas.lineage.enable.connection.lineage", false),
     ATLAS_INDEXSEARCH_ENABLE_JANUS_OPTIMISATION_EXTENDED("atlas.indexsearch.enable.janus.optimization.extended", false),
+    ATLAS_INDEXSEARCH_ENABLE_BULK_FETCHING("atlas.indexsearch.bulk.fetching.enable", false),
+    ATLAS_INDEXSEARCH_BULK_FETCHING_SIZE("atlas.indexsearch.bulk.fetching.size", 100),
```

Comment:
Create with different name, might be like INDEXSEARCH_EDGE_BULK_FETCH_BATCH_SIZE and lets set it 10